doc/manual/tuto.md

```diff
@@ -0,0 +1,179 @@
+---
+slug: tutorial
+---
+
+# Managing users
+
+In this tutorial, you will learn how use Nickel to manage a list of users,
+and then export it as a YAML file for a program consuming it.
+
+## Step 1: Install nickel
+
+The first step is to obtain nickel on your system. There are different
+methods to install Nickel, they are all covered on the [Getting
+started](https://nickel-lang.org/getting-started/#getting-started)
+page of Nickel project website.
+
+## Step 2: Think about the schema
```

Comment:
Maybe it's worth adding a paragraph explaining that writing a contract is optional. That is, you could just write out the users configuration as a list of records and export that. The reason to spend the time thinking about a contract is to catch mistakes like in step 6

Reply:
Good